canary/apps/react/vite/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "vite",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "start": "vite preview --port 3000"
+  },
+  "dependencies": {
+    "@aws-amplify/ui-react": "^2.7.0",
+    "aws-amplify": "^4.3.15",
```

Comment:
```suggestion
    "aws-amplify": "latest",
```

Reply:
Updated in 872efbc6a14eb22f5d3914d255ed12c7e740749f